core/src/main/java/org/apache/accumulo/core/iterators/WrappingIterator.java

```diff
@@ -58,21 +58,21 @@ protected void setSource(SortedKeyValueIterator<Key,Value> source) {
   public Key getTopKey() {
     if (seenSeek == false)
       throw new IllegalStateException("never been seeked");
-    return getSource().getTopKey();
+    return source.getTopKey();
```

Comment:
Interesting, the init method prevents making the source final.  Didn't see that one coming.